x/txfees/keeper/hooks.go

```diff
@@ -95,12 +212,10 @@ func (k Keeper) swapNonNativeFeeToDenom(ctx sdk.Context, denomToSwapTo string, f
 		// the next epoch.
 		poolId, err := k.protorevKeeper.GetPoolForDenomPairNoOrder(ctx, denomToSwapTo, coin.Denom)
 		if err != nil {
-			if err != nil {
-				// The pool route either doesn't exist or is disabled in protorev.
-				// It will just accrue in the non-native fee collector account.
-				// Skip this denom and move on to the next one.
-				continue
-			}
```

Comment:
I think this double err != nil was an accident?

Reply:
lol true